### DISCUSSION TITLE
Stack Stars/List additions

### DISCUSSION THREAD

**Discussion by twobiers** [community/community#20112] [Other Feature Feedback, Questions, & Ideas]
At the moment list additions are not stacked based on the list they are added to and each list addition results in a single entry within the feed.
It would be great if those additions are stacked like "NullDev added 3 repositories to Retro and Games"

![image](https://user-images.githubusercontent.com/22715034/177585419-a2421850-7951-4376-8932-1f160c7f74a4.png)

**Comment by siminapasat** [comment]
Thanks for your feedback @twobiers ! This is already on our roadmap, but we didn't get to it yet 😊

**Reply by radar-blip** [reply]
👍🏼
